tasks/hyrax-metadata-updates/tests/test-integration-index.js

```diff
@@ -0,0 +1,314 @@
+'use strict';
+
+const nock = require('nock');
+const { promisify } = require('util');
+const test = require('ava');
+const fs = require('fs');
+const xml2js = require('xml2js');
+
+const xmlParseOptions = {
+  ignoreAttrs: true,
+  mergeAttrs: true,
+  explicitArray: false
+};
+
+const { s3 } = require('@cumulus/aws-client/services');
+const {
+  randomId
+} = require('@cumulus/common/test-utils');
+const path = require('path');
+const {
+  buildS3Uri,
+  recursivelyDeleteS3Bucket,
+  promiseS3Upload,
+  parseS3Uri,
+  getS3Object
+} = require('@cumulus/aws-client/S3');
+const readFile = promisify(fs.readFile);
+const { InvalidArgument } = require('@cumulus/errors');
+const ValidationError = require('@cumulus/cmr-client/ValidationError');
+const { RecordDoesNotExist } = require('@cumulus/errors');
+
+const rewire = require('rewire');
+
+const HyraxMetadataUpdate = rewire('../index');
+
+const hyraxMetadataUpdate = HyraxMetadataUpdate.__get__('hyraxMetadataUpdate');
+const generateHyraxUrl = HyraxMetadataUpdate.__get__('generateHyraxUrl');
+const generatePath = HyraxMetadataUpdate.__get__('generatePath');
+const getEntryTitle = HyraxMetadataUpdate.__get__('getEntryTitle');
+
+test.before(() => {
+  nock.disableNetConnect();
+  nock.enableNetConnect('127.0.0.1');
+});
+
+test.beforeEach(async () => {
```

Comment:
Does this function need to be `async`? I'm not very experienced with `nock` but it doesn't look like there is anything being `await`ed.